cmd/gke-gcloud-auth-plugin/gcloud_token_provider.go

```diff
@@ -45,13 +48,13 @@ func (p *gcloudTokenProvider) readGcloudConfig() (*gcloudConfiguration, error) {
 }
 
 func (p *gcloudTokenProvider) token() (string, *time.Time, error) {
+	gcloudArgs := p.getGcloudArgs()
```

Comment:
Please move this line down closer to the usage and avoiding interfering with cloudsdkAuthAccessToken...